src/components/MapView/DateSelector/index.tsx

```diff
@@ -53,8 +53,8 @@ function DateSelector({ availableDates = [], classes }: DateSelectorProps) {
     setSelectedDate(moment(stateStartDate));
   }, [stateStartDate]);
   function updateMonth(month: number) {
-    const { startDate, endDate } = getMonthStartAndEnd(month, selectedYear);
-    dispatch(updateDateRange({ startDate, endDate }));
+    const dates = findAvailableDayInMonth(month, selectedYear, availableDates);
+    dispatch(updateDateRange({ startDate: dates[0] }));
```

Comment:
@dubwalla do we still not use `endDate` for anything? (Noted [here](https://github.com/WFP-VAM/prism-frontend/blob/0ab5eb16973d5b45dbe55cb2f408a2e4d239abe5/src/context/mapStateSlice/index.ts#L9)). Might be worthwhile to just remove to make things simpler.

Reply:
Let's keep it. This came up again last week and will need to be implemented soon-ish